internal/models/validations.go

```diff
@@ -0,0 +1,116 @@
+package models
```

Comment:
IMO models is not the best name as we have models package from swagger

Reply:
@tsorya actually initially I thought that these types belong into that "models" package generated fro the OpenAPI specification. But the "ValidationsInfo" of the "Host" type is a string containing a JSON document instead of a typed object. I don't know that is the reason for that, but I assume there is one. That makes me think that we consider that representation of validation results an "internal model", thus the name of the package. Note that in the package selector used in the rest of the code is "internalmodels" to avoid the conflict.

What is your suggestion for the name of this package?